configure.in

```diff
@@ -86,6 +101,23 @@ SAVE_CFLAGS="$CFLAGS"
 AC_PROG_CC([$($PG_CONFIG --cc)])
 CFLAGS="$SAVE_CFLAGS"
 
+host_guess=`${SHELL} $srcdir/config/config.guess`
```

Comment:
1. How does PostgreSQL get away without having `config.guess`?
2. Can't we just use something like `uname -mo`?